src/main/java/com/echobox/api/linkedin/connection/versioned/VersionedVideoConnection.java

```diff
@@ -75,7 +76,26 @@ public VersionedVideoConnection(VersionedLinkedInClient linkedinClient) {
     super(linkedinClient);
   }
   
-  public URN uploadVideo(InitializeUploadRequest initializeUploadRequest, String filePath)
+  public URN uploadVideoFromURL(InitializeUploadRequest initializeUploadRequest, String videoURL)
```

Comment:
This method looks very similar to `uploadVideoFromFile`. I'd suggest to extract out the common functionality into it's own method.